test/data/users.js

```diff
@@ -27,3 +27,11 @@ const testData = [
     "friends": []
   }
 ];
+
+// module.exports = {
+//   testData
+// }
+
+if (typeof module !== 'undefined') {
+   module.exports = testData;
```

Comment:
I think testData needs to be wrapped in {} to make it an object so we can access the properties

Reply:
I believe it is. It is currently setup how you do and things are working

Reply:
I went in to the resolve conflicts area and went with what we had on the master as it was working properly